web/src/components/Navigation/ProfileInfo/ProfileInfo.tsx

```diff
@@ -67,22 +77,25 @@ const ProfileIcon: React.FC = () => {
     <Popover>
       {({ isOpen, close: closePopover }) => (
         <React.Fragment>
-          <PopoverTrigger
-            as={AbstractButton}
-            display="flex"
-            width={40}
-            height={40}
-            backgroundColor={isOpen ? 'violet-400' : 'violet-500'}
-            _hover={{ backgroundColor: 'violet-400' }}
-            transition="background-color 0.1s linear"
-            borderRadius="circle"
-            justifyContent="center"
-            alignItems="center"
-            fontSize="small"
-            fontWeight="bold"
-            aria-label="Toggle User Menu"
-          >
-            {getUserInitials(userInfo).toUpperCase()}
+          <PopoverTrigger as={AbstractButton} aria-label="Toggle User Menu">
+            <Flex spacing={2} align="center">
+              <Box
+                width={40}
+                display="flex"
+                transition="background-color 0.1s linear"
+                borderRadius="circle"
+                justifyContent="center"
+                alignItems="center"
+                fontSize="small"
+                fontWeight="bold"
+                height={40}
+                backgroundColor={isOpen ? 'violet-400' : 'violet-500'}
+                _hover={{ backgroundColor: 'violet-400' }}
```

Comment:
To make sure it never becomes an "ellipsis" during overflow. It always stays a "perfect circle"

```suggestion
                _hover={{ backgroundColor: 'violet-400' }}
                flexShrink={0}
```